pwndbg/dbg/lldb/__init__.py

```diff
@@ -682,6 +682,12 @@ def alive(self) -> bool:
             and self.process.GetState() != lldb.eStateDetached
         )
 
+    @override
+    def stopped_with_signal(self) -> bool:
+        return self.process.GetState() == lldb.eStateStopped and any(
+            (thread.GetStopReason() == lldb.eStopReasonSignal for thread in self.process.threads)
```

Comment:
Hmm... Can't we get the current stopped thread stop reason instead of checking all of them in LLDB?